packages/server/tests/acceptance/equivalence.spec.ts

```diff
@@ -0,0 +1,884 @@
+    let contractCallResult;
+    try {
+      contractCallResult = await servicesClient.executeContractCallWithAmount(
+        ETH_PRECOMPILE_0x1001,
+        NON_EXISTING_FUNCTION,
+        EMPTY_FUNCTION_PARAMS,
+        1_000_000,
+        100,
+      );
+    } catch (e) {
+      const contractActions = await getContractActions(getTransactionIdFromException(e));
+      assert.fail(`${e.message}\ncontact actions:\n${JSON.stringify(contractActions, null, 2)}`);
+    }
+
+    const record = await getResultByEntityIdAndTxTimestamp(
+      ETH_PRECOMPILE_0x1001,
+      contractCallResult.contractExecuteTimestamp,
+    );
+    validateContractCall(record, ETH_PRECOMPILE_0x1001, SUCCESS, CONTRACT_EXECUTION_EXCEPTION);
+  });
+
+  // EQV-13 This should be successful due to hollow account creation
+  // Skipped as test is not ready
+  it.skip('should execute direct call to address over 1000 with amount - case hollow acconut', async function () {
+    const hollowAccount = ethers.Wallet.createRandom();
+    // convert evm address to hedera address
+    const hollowAcAddress = hollowAccount.address.toString();
+    let contractCallResult;
+
+    try {
+      contractCallResult = await servicesClient.executeContractCallWithAmount(
+        hollowAcAddress,
+        NON_EXISTING_FUNCTION,
+        EMPTY_FUNCTION_PARAMS,
+        1_000_000,
+        100,
+      );
+    } catch (e) {
+      const contractActions = await getContractActions(getTransactionIdFromException(e));
+      assert.fail(`${e.message}\ncontact actions:\n${JSON.stringify(contractActions, null, 2)}`);
+    }
+
+    const record = await getResultByEntityIdAndTxTimestamp(
+      ETH_PRECOMPILE_0x1001,
+      contractCallResult.contractExecuteTimestamp,
+    );
+    validateContractCall(record, ETH_PRECOMPILE_0x1001, SUCCESS, CONTRACT_EXECUTION_EXCEPTION);
+  });
+
+  [
+    {
+      // Equivalence 014, 040, 052
+      callTypes: [CallTypes.Call, CallTypes.StaticCall, CallTypes.DelegateCall],
+      addresses: [ADDRESS_0_0_0],
+      amount: 0,
+      outcome: Outcomes.Output,
+      message: '0x',
+    },
+    {
+      // Equivalence-015
+      callTypes: [CallTypes.Call],
+      addresses: [ADDRESS_0_0_0],
+      amount: 100,
+      outcome: Outcomes.Error,
+      message: INVALID_FEE_SUBMITTED,
+    },
+    {
+      // Equivalence-020
+      callTypes: [CallTypes.Call],
+      addresses: ['0.0.1', '0.0.2', '0.0.3', '0.0.4', '0.0.5', '0.0.6', '0.0.7', '0.0.8', '0.0.9'],
+      amount: 100,
+      outcome: Outcomes.Error,
+      message: INVALID_FEE_SUBMITTED,
+    },
+    {
+      // Equivalence-022
+      callTypes: [CallTypes.Call],
+      addresses: ['0.0.10', '0.0.100', '0.0.358'],
+      amount: 100,
+      outcome: Outcomes.Error,
+      message: INVALID_FEE_SUBMITTED,
+    },
+    {
+      // Equivalence-021, 045, 057; Address 0.0.10 currently skipped because of ongoing issue being investigated
+      callTypes: [CallTypes.Call, CallTypes.StaticCall, CallTypes.DelegateCall],
```

Comment:
Extracted above the test body as 'basicTests' (commonTests). Regarding exporting them in separate file it would reduce visibility and I would advice against it except a case where those exact same tests will be reusable elsewhere.